website/docs/secrets/manage-secrets-gui.mdx

```diff
@@ -3,18 +3,131 @@ title: Manage Secrets GUI
+
+
+### External Secret Details
+
+![External Secret Details](./imgs/eso-details-1.png)
+
+The details page displays the details of a specific external secret, including its name, namespace, data, and creation date. Below are the details that you can expect to see on this page:
+
+- **Status:** This indicates the current status of the external secret, which can be "Ready" or "Not Ready" depending on whether the external secret has been successfully created and is ready for use.
+
+- **Last Updated:** This shows the date and time when the external secret was last updated.
+
+- **External Secret:** This is the name of the external secret that you are viewing.
+
+- **K8s Secret:** This is the name of the Kubernetes secret that is associated with the external secret.
+
+- **Cluster:** This indicates which cluster the external secret exists on.
+
+- **Secret Store:** This shows the name of the secret store provider that is being used to store the external secret.
+
+- **Secret Store Type:** This indicates the type of secret store that is being used to store the external secret. In this case, the type is "AWS Secrets Manager".
+
+- **Secret path:** This is the path to the external secret within the secret store.
+
+- **Property:** This is the property or key that is associated with the external secret.
+
+- **Version:** This shows the version of the external secret, which may be blank if no version has been specified.
+
+Understanding the information provided on the details page can help you to manage and troubleshoot your external secrets as needed.
+
+
+### Understanding Events
+
+![External Secret Events](./imgs/external-secret-events-1.png)
+
+The following events can be expected when using the UI to manage external secrets:
+
+- **Updated:** This event indicates that an existing external secret has been successfully updated with new data.
+
+- **Not Ready:** This event indicates that there was an issue with the secret store when trying to access or synchronize external secrets. This includes situations such as the secret store being unavailable or not ready to handle requests, or issues with authentication when trying to access the secret store.
+
+Understanding these events can help you to troubleshoot issues that may arise when managing external secrets using the UI. In particular, if you encounter a `Not Ready` event, you may need to check your secret store credentials and ensure that the secret store is operational before proceeding with any further actions.
+
+
+## SOPS
+
+### Getting Started with SOPS
+
+[SOPS](https://github.com/mozilla/sops#usage) provides an encryption mechanism for kubernetes secrets that are stored in Git repositories.
+
+### Prerequisites
+
+Setup the SOPS operator by following [this](//setup-sops-page-url) guide.
+
+### Create SOPS Secret
+
```

Comment:
Let's add a small paragraph on what creating SOPS secret is. i.e it encrypts the secret using public key secret on the cluster.
then let's add a section (or reference) on how to make the public key available in the kustomization list.